nixos/modules/config/xdg/mime.nix

```diff
@@ -0,0 +1,31 @@
+{ config, lib, pkgs, ... }:
```

Comment:
Aha, good catch. I had thought I could get away with removing them since the use of the binary is now encoded here, but it seems like it also has data to install.